### ISSUE TITLE
Captchas never generated

### ISSUE THREAD

**Issue by jacobeva** [KushBlazingJudah/fedichan#2]
Currently, when attempting to perform an action which requires completing a captcha, it never succeeds, as no captcha has been generated and served to the user.

Not sure why they're not generating, I don't have any errors in my logs besides the following one strangely.

```
routes/util.go:38:populateCaptcha() sql: no rows in result set
```

**Comment by jacobeva** [comment]
I'm on Arch.

**Comment by anomalous69** [comment]
> I'm on Arch.

Do you have imagemagick installed and is the `convert` executable in your PATH.
Testing on Arch Linux now and I noticed the server doesn't actually provide an error if it can't find/run convert, other than routes/util.go:38:populateCaptcha() sql: no rows in result set.

Here is the list of packages I needed to get it working on Arch.
`pacman -S postgresql go exiv2 imagemagick gsfonts git make`

**Comment by jacobeva** [comment]
Yeah, I have ImageMagick installed. I think you're onto something here, I reckon I was missing `gsfonts` from my system. Unable to test currently, will report back in the coming days.

**Comment by KushBlazingJudah** [comment]
0bacca7 should log something in the event it cannot execute `convert`. If there is nothing, then `convert` should've complained in stderr. My desktop system, running Alpine Linux edge, works almost (thin characters, otherwise fine) out of the box and I didn't have `ghostscript-fonts` installed, maybe it's something else?
I faintly recall needing `font-opensans` (probably `ttf-opensans` elsewhere) in the past but it seems like I don't need it anymore?
